components/logins/ios/Logins/LoginsStorage.swift

```diff
@@ -94,26 +135,53 @@ open class LoginsStorage {
         }
     }
 
+    /// Unlock the database.
+    /// `key` can be any string as it is a passphrase (e.g. `bobo`).
+    /// `salt` must be an hex-encoded string of 32 characters (e.g. `a6a97a03ac3e5a20617175355ea2da5c`).
```

Comment:
What happens if `salt` is not such a thing, will we throw an error?

Reply:
Added a check and an error type.